src/mono/wasm/Directory.Build.props

```diff
@@ -0,0 +1,3 @@
+<Project>
```

Comment:
why are those necessary? MSBuild searches through the dir tree to find the nearest D.B.props & targets.

Reply:
ah yeah this was from an earlier iteration where I didn't use the `Microsoft.Build.NoTargets` in wasm.proj, will remove.